Jenkinsfile

```diff
@@ -19,30 +18,11 @@ node {
       maxConcurrentTotal: 0,
       paramsToUseForLimit: 'release',
       throttleEnabled: true,
-      throttleOption: 'category'],
-    [$class: 'ParametersDefinitionProperty',
-      parameterDefinitions: [
-        [$class: 'BooleanParameterDefinition',
-          name: 'IS_SCHEMA_TEST',
-          defaultValue: false,
-          description: 'Identifies whether this build is being triggered to test a change to the content schemas'],
-        [$class: 'StringParameterDefinition',
-          name: 'SCHEMA_BRANCH',
-          defaultValue: DEFAULT_SCHEMA_BRANCH,
-          description: 'The branch of govuk-content-schemas to test against']]
+      throttleOption: 'category'
```

Comment:
This option isn't needed and if it were it usually has a parameter (for the category to throttle on)